### DISCUSSION TITLE
kairship/instance/add

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2282] [Announcements]
# kairship/instance/add

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/instance/add.html

**Comment by liyuerich** [comment]
若安装失败，可以在对应的管理面集群中查看原因。 

这个察看原因和实际的情况已经不符合了，可以更新文档

**Reply by windsonsea** [reply]
Hope this was fixed in PR #5312
